src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs

```diff
@@ -325,5 +326,15 @@ private void EnterOutliningMode_Unchecked(object sender, RoutedEventArgs e)
             Collapse_metadata_signature_files_on_open.IsEnabled = false;
             Collapse_sourcelink_embedded_decompiled_files_on_open.IsEnabled = false;
         }
+
+        private void Run_code_analysis_in_separate_process_Checked(object sender, RoutedEventArgs e)
+        {
+            Run_code_analysis_on_dotnet.IsEnabled = true;
+        }
+
+        private void Run_code_analysis_in_separate_process_Unchecked(object sender, RoutedEventArgs e)
+        {
+            Run_code_analysis_on_dotnet.IsEnabled = false;
+        }
```

Comment:
📝 I'd prefer to not do this

Reply:
I added it just in case. We can always remove it in before GA